### ISSUE TITLE
ModuleNotFoundError: No module named 'osgeo' on Arch/Parch linux

### ISSUE THREAD

**Issue by tomachinz** [qgis/QGIS#59786]
### What is the bug or the crash?

Yesterday I updated my os-release file - from Parch to Arch it seems to have helped some installs like webmin.

I'm running "Parch" distro which is a very of Arch Linux.

cat /etc/os-release
DISTRIB_ID="Arch"
DISTRIB_RELEASE="rolling"
DISTRIB_DESCRIPTION="Arch Linux"
putin# ls -la /etc/os-release 
-rw-r--r-- 1 root root 77 Dec  8 22:40 /etc/os-release

Installed using 
`paru -S qgis`

Upon launching from desktop entry at /usr/share/applications I see:

```
An error occurred during execution of following code:
from osgeo import gdal, ogr, osr


Traceback (most recent call last):
  File "", line 1, in 
  File "/usr/lib/python3.12/site-packages/qgis/utils.py", line 966, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'osgeo'


Python version:
3.12.7 (main, Oct  1 2024, 11:15:50) [GCC 14.2.1 20240910]

QGIS version:
3.40.1-Bratislava 'Bratislava', exported

Python path:
['/usr/share/qgis/python', '/home/tom/.local/share/QGIS/QGIS3/profiles/default/python', '/home/tom/.local/share/QGIS/QGIS3/profiles/default/python/plugins', '/usr/share/qgis/python/plugins', '/usr/lib/python312.zip', '/usr/lib/python3.12', '/usr/lib/python3.12/lib-dynload', '/home/tom/.local/lib/python3.12/site-packages', '/usr/lib/python3.12/site-packages', '/home/tom/.local/share/QGIS/QGIS3/profiles/default/python']

```

Not sure if this is useful to anybody working on the project? 
I am not a heavy GIS user and just starting to tinker with this stuff more seriously for a startup idea I had.
Came to Arch from Kubuntu prefer not to use "snap" etc

**Could this be caused by stale and old user config? I am running from a mounted /home partition which may have junk.
I am open to deleting /home/tom/.local/share/QGIS and verify my python install not sure how. 

### Steps to reproduce the issue

paru -S qgis 
on an install of "Parch" linux

### Versions

QGIS version
3.40.1-Bratislava
QGIS code branch
Release 3.40
 
Libraries
Qt version
5.15.16
Python version
3.12.7
GDAL/OGR version
3.10.0
PROJ version
9.5.0
EPSG Registry database version
v11.016 (2024-08-31)
GEOS version
3.13.0-CAPI-1.19.0
SQLite version
3.46.1 (Compiled)
3.47.1 (Running)
PDAL version
2.8.0 (Compiled)
2.8.2 (Running)
PostgreSQL client version
16.3
SpatiaLite version
5.1.0
QWT version
6.3.0
QScintilla2 version
2.14.1
OS version
Linux 6.12.3-arch1-1

### Supported QGIS version

- [X] I'm running a supported QGIS version according to [the roadmap](https://www.qgis.org/en/site/getinvolved/development/roadmap.html#release-schedule).

### New profile

- [ ] I tried with a new [QGIS profile](https://docs.qgis.org/latest/en/docs/user_manual/introduction/qgis_configuration.html#working-with-user-profiles)

### Additional context

I will torch that user profile now...

**Comment by alexbruy** [comment]
This is not a QGIS issues, but a packaging/installation problem in your distro. Also note that you are using AUR helper not the OS package manager, this may be another source of problems.

**Comment by tomachinz** [comment]
Thanks. It does eventually load. I will try the release version 

![image](https://github.com/user-attachments/assets/c543b01e-beaa-4d03-82b1-c559a2b37025)
The current available LTR version of QGIS is 3.34.14. 
The latest available version of QGIS is 3.40.2. 
We are working on 3.41-Master. It will go into feature freeze on 2025-01-17 12:00:00 UTC and be released as 3.42 on 2025-02-21 12:00:00 UTC. 

I will try install via pacman next time cheers.
